ww4ff/run/train.py

```diff
@@ -99,6 +109,8 @@ def evaluate(dataset: WakeWordEvaluationDataset, prefix: str):
             loss.backward()
             optimizer.step()
             pbar.set_postfix(dict(loss=f'{loss.item():.3}'))
+        for group in optimizer.param_groups:
+            group['lr'] *= 0.75
```

Comment:
we decrease every iteration?

Reply:
Yeah, every epoch for a simple exponential LR decrease. We can probably switch to the plateau one later.